mx.sulong/suite.py

```diff
@@ -215,6 +215,14 @@
       ],
       "sha1" : "5552052e53fcd7fe40c558866c9cd51027c17322",
     },
+    "LLVMTESTSUITE" : {
+      "path" : "tests/test-suite-3.2.src.tar.gz",
+      "urls" : [
+        "https://lafo.ssw.uni-linz.ac.at/pub/sulong-deps/test-suite-3.2.src.tar.gz",
+        "http://llvm.org/releases/3.2/test-suite-3.2.src.tar.gz",
```

Comment:
The `http` link is okay in this situation, as mx verifies the sha1 hash of the download. I will still remove the `http` link as we don't really need it.